src/Data/API/Tools/CBOR.hs

```diff
@@ -170,15 +170,15 @@ encodeUTCTime :: UTCTime -> Encoding
 #if MIN_VERSION_aeson(0,8,1)
 encodeUTCTime = encodeUTCTime'
 #else
-encodeUTCTime t = encodeTag 0 <> encode (formatTime defaultTimeLocale fmt t)
+encodeUTCTime t = encode (parseUTC_ $ formatTime defaultTimeLocale fmt t)
```

Comment:
I don't buy this. It uses `encode` at type `Maybe UTCTime`, which doesn't seem right.

How about we drop support for `aeson < 0.8.1`, get rid of all this complexity and the CPP conditionals?

Reply:
Right, forgot to deal with the Maybe and types didn't help this one time. :) Will rip out the old aeson shims instead.